Source/Utilities/DOMReady.js

```diff
@@ -77,6 +77,7 @@ if ('onreadystatechange' in document) document.addListener('readystatechange', c
 else shouldPoll = true;
 
 if (shouldPoll) poll();
+if (!ready) check();
```

Comment:
@arian good point. Did some more tests in Travis and in a IE8 VM using:

```
if (shouldPoll) poll();
checks.each(function(chk){
	alert(chk + '');
});
if (!ready) check();
```

and all cases showed the `document.readyState` test function is tested first. 
I tested also removing that IE8 block [on Travis](https://travis-ci.org/SergioCrisostomo/mootools-core/builds/47588654) and it passed in all Browsers. 
It looks clean in that way.

We could import some more tests from [the old PHP file](https://github.com/mootools/mootools-core/blob/master/Tests/DOMReady.php) if it feels like we are missing something, since the IE8 block seems redundant to these specs.